src/routers/sessions.py

```
@@ -212,23 +226,63 @@ async def update_session(
         raise ResourceNotFoundException("Session not found") from e
 
 
+async def _delete_session_background_task(workspace_id: str, session_id: str):
```

Ruff (0.14.5)

Missing return type annotation for private function `_delete_session_background_task`

Add return type annotation: `None`

(ANN202)